osfoffline/settings/defaults.py

```diff
@@ -47,3 +47,9 @@
 
 PROJECT_LOG_DIR = user_log_dir(appname=PROJECT_NAME, appauthor=PROJECT_AUTHOR)
 PROJECT_LOG_FILE = os.path.join(PROJECT_LOG_DIR, 'osfoffline.log')
+
+#updater
+REPO = 'CenterForOpenScience/OSF-Offline'
+VERSION = '0.2.1'
+NAME = 'OSF-Offline'
+MIN_VERSION_URL = 'https://raw.githubusercontent.com/CenterForOpenScience/OSF-Offline/develop/deploy/Offline-version.json'
```

Comment:
> This URL will only exist if this pr got merged

On a related note: do you have any notes that you used, describing exactly how you tested this PR when you ran the code?

(this will help QA to determine whether the implementation meets the intended requirements)